docs/integration/skills.md

```
@@ -341,3 +341,248 @@ Enable debug logging:
+# Interactive login (prompts for email and password)
+datus skill login --marketplace http://my-town:9000
+
+# Non-interactive login
+datus skill login --marketplace http://my-town:9000 --email user@example.com --password secret
+
+# Logout (clear saved token)
+datus skill logout --marketplace http://my-town:9000
+```
+
+In the REPL:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)